vnext/Shared/DevServerHelper.h

```diff
@@ -32,14 +32,16 @@ class DevServerHelper {
       const std::string &jsbundle,
       const std::string &platform,
       const std::string &dev,
-      const std::string &hot) {
+      const std::string &hot,
```

Comment:
If you're here, want to fix this typo 😛 .

Reply:
Not sure what typo you're mentioning, but refactored these 2 to pass as bools too.

---
In reply to: [467328877](https://github.com/microsoft/react-native-windows/pull/5666#discussion_r467328877) [](ancestors = 467328877)

Reply:
Oh whoops, thought this was supposed to be host. Got tripped up by string vs bool.